fabric-registry-sync-v0/build.gradle

```diff
@@ -13,4 +13,5 @@ moduleDependencies(project, [
 testDependencies(project, [
 	':fabric-lifecycle-events-v1',
 	':fabric-command-api-v2',
+	':fabric-resource-loader-v0',
```

Comment:
I think it's already included by default for all modules

Reply:
TIL, I'll remove it from here then.